egs2/tamil/asr1/local/score.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env bash
```

Comment:
Hi @SujaySKumar , can you instead use the common scoring file https://github.com/espnet/espnet/blob/master/egs2/TEMPLATE/asr1/pyscripts/utils/score_intent.py?

Reply:
Done